motile/utils.py

```diff
@@ -1,57 +0,0 @@
-import networkx as nx
```

Comment:
this file continues to go out of sync with the track_graph, and isn't tested anywhere... so I just deleted it for now.  If you'd prefer to comment it all out until it gets some love, I can do that instead (but, it's always in git history too, so I have slight preference for deleting)